src/test/java/uk/gov/hmcts/ccd/domain/service/createcase/SubmitCaseTransactionTest.java

```diff
@@ -1,7 +1,5 @@
 package uk.gov.hmcts.ccd.domain.service.createcase;
 
```

Comment:
wonder whether we should have some integration tests to verify that the writing to the 2 tables case_event and the new table is atomic. i.e. if one fails the other is rolledback

Reply:
done